src/main/java/com/bpodgursky/jbool_expressions/ExprUtil.java

```diff
@@ -60,11 +78,11 @@
 
   public static <K> Expression<K>[] allExceptMatch(List<Expression<K>> exprs, Expression<K> omit) {
     //noinspection unchecked
-    return allExceptMatch(exprs.toArray(new Expression[0]), omit);
+    return allExceptMatch(exprs.toArray(new Expression[exprs.size()]), omit);
```

Comment:
technically this is exprs.size()-1 I believe

Reply:
While inefficient this is correct:
exprs.toArray(new Expression[exprs.size()]) is converting the list to an array. If we set this to one less then it'll have to resize immediately.